src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -428,26 +432,21 @@ public JSONObject getStats()
             {
                 confSize = 1;
             }
+
             numParticipants += confSize;
+            endpointPairs += confSize * confSize;
             largestConferenceSize = Math.max(largestConferenceSize, confSize);
 
-            int conferenceSizeIndex = confSize < conferenceSizes.length
-                    ? confSize
-                    : conferenceSizes.length - 1;
-            conferenceSizes[conferenceSizeIndex]++;
+            conferenceSizes.addValue(confSize);
 
             jibriRecordersAndGateways.add(conference.getJibriRecorder());
             jibriRecordersAndGateways.add(conference.getJibriSipGateway());
         }
 
         stats.put("largest_conference", largestConferenceSize);
         stats.put("participants", numParticipants);
-        JSONArray conferenceSizesJson = new JSONArray();
-        for (int size : conferenceSizes)
-        {
-            conferenceSizesJson.add(size);
-        }
-        stats.put("conference_sizes", conferenceSizesJson);
+        stats.put("conference_sizes", conferenceSizes.toJson());
+        stats.put("endpoint_pairs", endpointPairs);
```

Comment:
I'm not sure about "endpoint_pairs" as the name of this.  What's the motivation for this?

Reply:
To see how this number correlates with load on the machine or the prosody process (in order to potentially use it to set the shard's weight for load balancing)